Gamma_Analysis.py

```diff
@@ -0,0 +1,341 @@
+    right_peak = energy + 2*FWHM
+
+    right_start = 0
+    while energy_axis[right_start] < (right_peak - FWHM):
+        right_start += 1
+
+    right_end = right_start
+    while energy_axis[right_end] < (right_peak + FWHM):
+        right_end += 1
+
+    gross_counts_right = sum(M_counts[right_start:right_end])
+
+    # Net Area
+
+    net_area = gross_counts_peak - (gross_counts_left + gross_counts_right)/2
+
+    # Uncertainty
+
+    uncertainty = abs((gross_counts_peak +
+                      (gross_counts_left + gross_counts_right) / 4)) ** 0.5
+
+    # Returning results
+
+    results = [net_area, uncertainty]
+
+    return results
+
+
+def Background_Subtract(M, B):
+    """
+    Background_Subtract will subtract the background spectrum
+    from a given measurement. This will return the energy bins using a given
+    energy calibration as well as the background subtracted counts of a
+    measurement. Since both spectra come from the same detector, this assumes
+    the energy calibrations are the same.
+    """
+
+    M_Counts = M.data
+    B_Counts = B.data
+
+    M_Time = M.livetime
+    B_Time = B.livetime
+
+    B_Channels = B.channel
+    E0 = M.energy_cal[0]
+    Eslope = M.energy_cal[1]
+
+    Energy_Axis = M.channel
+    channel_ratio = (len(Energy_Axis)+1)/(len(B_Channels)+1)
+    if channel_ratio == 1:
+        Energy_Axis = Energy_Axis.astype(float)
+        Energy_Axis[:] = [E0+Eslope*x for x in B_Channels]
+        B_Counts_M = [1.0]*len(M.channel)
+        B_Counts_M[:] = [x*(M_Time/B_Time) for x in B_Counts]
+        M_Sub_Back = [1.0]*len(B_Channels)
+        M_Sub_Back = [M_Counts[x]-B_Counts_M[x] for x in M.channel]
+        Sub_Spect = [Energy_Axis, M_Sub_Back]
+    else:
+        MOD = (len(Energy_Axis)+1) % channel_ratio
+        REM = channel_ratio - MOD + 1
+        M_Counts = np.append(M_Counts, np.zeros(REM))
+        M_Counts = M_Counts.reshape((-1, channel_ratio))
+        M_Counts = np.sum(M_Counts, 1)
+        Energy_Axis = Energy_Axis.astype(float)
+        Energy_Axis[:] = [E0+Eslope*x for x in M.channel]
+        Energy_Axis = np.append(Energy_Axis, np.zeros(REM))
+        Energy_Axis = Energy_Axis.reshape((-1, channel_ratio))
+        Energy_Axis = np.mean(Energy_Axis, 1)
+        B_Counts_M = [1.0]*len(B.channel)
+        B_Counts_M[:] = [x*(M_Time/B_Time) for x in B_Counts]
+        M_Sub_Back = [1.0]*len(M_Counts)
+        M_Sub_Back = [M_Counts[x]-B_Counts_M[x] for x in B.channel]
+        Sub_Spect = [Energy_Axis, M_Sub_Back]
+    return Sub_Spect
+
+
+def Bias_Check(Spectrum, Background, Measurement_Name):
+    """
+    Bias_Check will determine if there's a bias in the measurement. It will
+    look at high energy peaks that correspond to Bi-214 and Tl-208 since these
+    peaks occur in the background. If the net area of those peaks are negative
+    beyond 2 sigma, where sigma is the uncertainty of a peak net area, then a
+    message indicating a measurement bias will be produced.
+    """
```

Comment:
Again, add comments throughout so that someone can understand what the code is doing. The docstring here tells me the purpose or outcome of the function, but I don't follow your method.